client/reader/get-helpers.js

```diff
@@ -68,7 +69,13 @@ export const getSiteName = ( { feed, site, post } = {} ) => {
 };
 
 export const getSiteDescription = ( { site, feed } ) => {
-	return decodeEntities( ( site && site.description ) || ( feed && feed.description ) );
+	const description = decodeEntities(
```

Comment:
do we need the decode call anymore? I think this is decoded already in the reducer? 

https://github.com/Automattic/wp-calypso/blob/master/client/state/reader/feeds/reducer.js#L70

ahhhh

https://github.com/Automattic/wp-calypso/blob/master/client/state/reader/sites/reducer.js#L60

we don't do it for site objects. maybe we should?